app/assets/v2/js/grants/cart/rsk_extension.js

```diff
@@ -19,15 +19,13 @@ const contributeWithRskExtension = async(grant, vm, modal) => {
     try {
       console.log(ethereum.selectedAddress);
     } catch (e) {
-      modal.closeModal();
       _alert({ message: 'Please download or enable Nifty Wallet extension' }, 'danger');
       return;
     }
 
     if (!ethereum.selectedAddress) {
-      modal.closeModal();
       return onConnect().then(() => {
-        modal.openModal();
+        console.log('wallet connected');
```

Comment:
did we intend on changing this? not sure why remove the modal instance

Reply:
this was a bug as `modal` isn't passed as an input parameter in grants extension (only in bounty)